sky/packages/sky/lib/src/widgets/scrollable.dart

```diff
@@ -379,6 +379,34 @@ class Block extends StatelessComponent {
   }
 }
 
+abstract class ScrollingListPainter extends Painter {
+  RenderBlockViewport get renderer => renderObject as RenderBlockViewport;
```

Comment:
In general, if you ever find yourself using 'as' then there's likely a problem. Please don't hesitate to ping me if you feel the urge to use 'as'.